src/components/RetryFailedPayment.tsx

```diff
@@ -2,18 +2,21 @@ import { useEffect } from "react";
 import { Banner, Icon, Link, Loader } from "@stellar/design-system";
 import { useQueryClient } from "@tanstack/react-query";
 import { usePaymentsRetry } from "apiQueries/usePaymentsRetry";
-import { PaymentDetailsStatusHistoryItem } from "types";
+import { PaymentStatus } from "types";
 
 interface RetryFailedPaymentProps {
   paymentId: string;
-  paymentStatus: PaymentDetailsStatusHistoryItem[];
+  paymentStatus: PaymentStatus;
 }
 
 export const RetryFailedPayment = ({
   paymentId,
   paymentStatus,
 }: RetryFailedPaymentProps) => {
-  const isFailed = paymentStatus.slice(-1)[0]?.status === "FAILED";
+  console.log("paymentStatus", paymentStatus);
+  const isFailed = paymentStatus === "FAILED";
+
+  console.log("isFailed", isFailed);
```

Comment:
Do we need to keep `console.log` here?

Reply:
TY! removed in https://github.com/stellar/stellar-disbursement-platform-frontend/pull/72/commits/6ee56df35681fc75e960db532390db832f2fbfae